standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/leader/HouseKeepingTasks.java

```diff
@@ -114,6 +114,9 @@ public void lossLeadership(LeaderElection election) throws Exception {
       metastoreTaskThreadPool.shutdown();
       metastoreTaskThreadPool = null;
     }
+
+    HiveMetaStore.LOG.info("Stopped the {} Housekeeping tasks",
+        runOnlyRemoteTasks ? "remote" : "always");
```

Comment:
Where did we define what set of tasks are always run and what set of tasks runs remotely? I'm just curious to know if we have provided enough logging for the user to understand what stopped and what runs.

Reply:
Good suggestion, added the log for what stopped